src/graph/validator/MatchValidator.cpp

```diff
@@ -1292,19 +1292,25 @@ Status MatchValidator::validatePathInWhere(
 }
 
 /*static*/ Status MatchValidator::buildRollUpPathInfo(const MatchPath *path, Path &pathInfo) {
-  DCHECK(!DCHECK_NOTNULL(path->alias())->empty());
   for (const auto &node : path->nodes()) {
     // The inner variable of expression will be replaced by anno variable
     if (!node->alias().empty() && node->alias()[0] != '_') {
       pathInfo.compareVariables.emplace_back(node->alias());
     }
   }
   for (const auto &edge : path->edges()) {
-    if (edge->alias()[0] != '_') {
-      pathInfo.compareVariables.emplace_back(edge->alias());
+    const auto &edgeAlias = edge->alias();
+    if (!edgeAlias.empty() && edgeAlias.front() != '_') {
+      if (edge->range()) {
+        return Status::SemanticError(
+            "Variable '%s` 's type is list. not support used in multiple patterns "
```

Comment:
Use a friendlier error message: `Repeating variable length edge variables not supported`.

Reply:
i think it's ok

Reply:
Better to use `Edge List` instead of `list`.